includes/wc-conditional-functions.php

```diff
@@ -212,6 +212,22 @@ function is_order_received_page() {
 	}
 }
 
+if ( ! function_exists( 'is_payment_methods_page' ) ) {
```

Comment:
As per previous comment, lets add this to the is_add_payment_method_page function instead.